curriculum/challenges/english/25-front-end-development/lab-counting-cards/68a23004e96a105df3a21fc4.md

```diff
@@ -0,0 +1,158 @@
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab.
+
+**User Stories:**
+
+1. You should have a function called `cc`.
+1. The `cc` function should receive a `card` parameter which can either be a number or string.
+    - For values between `2` to `10`, the `card` parameter will be a number.
+    - For all other values, the `card` parameter will be a string.
+1. The `cc `function should modify the global `count` variable based on certain criteria.
+1. The global `count` variable should be increased by `1` for the cards `2`, `3`,`4`, `5`, or `6`
+1. The global `count` variable should remain unchanged for the cards `7`, `8`, `9`.
+1. The global `count` variable should be decreased for the cards `10`, `"J"`, `"Q"`, `"K"`, `"A"`
+1. The `cc` function should return a string with current count and the string `Bet` if the count is positive.
+1. The `cc` function should return a string with current count and the string `Hold` if the count is less than or equal to `0`.
+1. In the function output, the current count and the player's decision (`Bet` or `Hold`) should be separated by a space `-3 Hold`.
+
+# --hints--
+
+Your function should return the value of `count` and the text (`Bet` or `Hold`) with one space character between them.
+
+```js
+count = 0;
+let out = cc(10);
+const pattern = /^-?\d+ (Bet|Hold)$/;
+assert.match(out, pattern);
+```
+
+After the cards `2`, `3`, `4`, `5`, then calling `cc(6)` should return the string `5 Bet`.
+
+```js
+count = 0;
+cc(2);
+cc(3);
+cc(4);
+cc(5);
+const out = cc(6);
+assert.strictEqual(out, '5 Bet')
+```
+
+After the cards `7`, `8`, then calling `cc(9)` should return the string `0 Hold`.
+
+```js
+count = 0;
+cc(7);
+cc(8);
+const out = cc(9);
+assert.strictEqual(out, '0 Hold');
+```
+
+After the cards `10`, `"J"`, `"Q"`, `"K"`, then calling `cc('A')` should return the string `-5 Hold`.
+
+```js
+count = 0;
+cc(10);
+cc('J');
+cc('Q');
+cc('K');
+const out = cc('A');
+assert.strictEqual(out, '-5 Hold');
+```
+
+After the cards `3`, `7`, `"Q"`, `8`, then calling `cc('A')` should return the string `-1 Hold`.
+
+```js
+count = 0;
+cc(3);
+cc(7);
+cc('Q');
+cc(8);
+const out = cc('A');
+assert.strictEqual(out, '-1 Hold');
+```
+
+After the cards `2`, `"J"`, `9`, `2`, then calling `cc(7)` should return the string `1 Bet`.
+
+```js
+count = 0;
+cc(2);
+cc('J');
+cc(9);
+cc(2);
+const out = cc(7);
+assert.strictEqual(out, '1 Bet');
+```
+
+After the cards `2`, `2`, then calling `cc(10)` should return the string `1 Bet`.
+
+```js
+count = 0;
+cc(2);
+cc(2);
+const out = cc(10);
+assert.strictEqual(out, '1 Bet')
+```
+
+After the cards `3`, `2`, `"A"`, `10`, then calling `cc('K')` should return the string `-1 Hold`.
+
+```js
+count = 0;
+cc(3);
+cc(2);
+cc('A');
+cc(10);
+const out = cc('K');
+assert.strictEqual(out, '-1 Hold');
+```
+
+# --seed--
+
+## --seed-contents--
+
+
+```js
+let count = 0;
```

Comment:
Not a big deal, but these labs usually start without seed code. So it would make sense to remove `let count = 0` from the seed and require that in user stories.